rescript/plugin_setup.py

```diff
@@ -242,17 +241,17 @@
                 'taxonomy dataframe. "majority" finds the LCA consensus while '
                 'giving preference to majority labels. ' + super_lca_desc,
         'rank_handle_regex': rank_handle_description + rank_handle_extra_note,
-        'new_rank_handle': (
+        'new_rank_handles': (
             'Specifies the set of rank handles to prepend to taxonomic labels '
-            'at each rank. For example, "greengenes" will prepend 7-level '
-            'greengenes-style rank handles. Note that merged taxonomies will '
+            'at each rank. Note that merged taxonomies will '
             'only contain as many levels as there are handles if this '
-            'parameter is used. So "greengenes" will trim all taxonomies to '
-            'seven levels, even if longer annotations exist. Note that this '
+            'parameter is used. This will trim all taxonomies to the given '
+            'levels, even if longer annotations exist. Note that this '
             'parameter will prepend rank handles whether or not they already '
             'exist in the taxonomy, so should ALWAYS be used in conjunction '
             'with `rank_handle_regex` if rank handles exist in any of the '
-            'inputs.'),
+            'inputs. Use \'disable\' to prevent applying \'new_rank_handles\''
```

Comment:
applying --> appending

(just to make terminology consistent with the previous sentence)